ts/features/itwallet/credentials/store/reducers/index.ts

```diff
@@ -1,19 +1,25 @@
 import * as O from "fp-ts/lib/Option";
 import { getType } from "typesafe-actions";
 import { Action } from "../../../../../store/actions/types";
-import { itwCredentialsRemove, itwCredentialsStore } from "../actions";
+import {
+  itwCredentialsRemove,
+  itwCredentialsStore,
+  itwIpzsHasReadPolicy
+} from "../actions";
 import { StoredCredential } from "../../../common/utils/itwTypesUtils";
 import { CredentialType } from "../../../common/utils/itwMocksUtils";
 import { itwLifecycleStoresReset } from "../../../lifecycle/store/actions";
 
 export type ItwCredentialsState = {
   eid: O.Option<StoredCredential>;
   credentials: Array<O.Option<StoredCredential>>;
+  hasReadIPZSPolicy: boolean;
 };
 
 export const itwCredentialsInitialState: ItwCredentialsState = {
   eid: O.none,
-  credentials: []
+  credentials: [],
+  hasReadIPZSPolicy: false
```

Comment:
Question: why do we need to persist this data in this state slice? That's a special slice which is saved via secure storage.

Reply:
From what I could observe, the update of the properties is always done in bulk when the `updateMixpanelProfileProperties` function is called. I saved this value because, based on the previous implementation, we noticed that it was being reset to `false` when the update function is invoke after this value has been set.
We need to track this property with the correct value even if wallet instance has been activated, as it is not mandatory for the user to read those policies.